src/sentry/api/endpoints/organization_member/index.py

```diff
@@ -376,28 +376,32 @@ def post(self, request: Request, organization) -> Response:
         members_can_only_invite_to_members_teams = (
             not organization.flags.allow_joinleave and not organization.flags.disable_member_invite
         )
-        has_team_roles = "teamRoles" in result and bool(result["teamRoles"])
+        has_teams = bool(
+            ("teamRoles" in result and result["teamRoles"])
+            or ("teams" in result and result["teams"])
+        )
```

Comment:
i _think_ you can simplify to

```suggestion
        has_teams = bool(result.get("teamRoles") or result.get("teams"))
```

Reply:
ur right i forgot python 🫠